### ISSUE TITLE
Drop the "c" prefix

### ISSUE THREAD

**Issue by usepowershell** [ChrisAWalker/cSecurityOptions#5]
The "c" prefix is meaningless.

http://stevenmurawski.com/powershell/2015/06/dsc-people-lets-stop-using-c-now/

**Comment by ChrisAWalker** [comment]
@smurawski - I agree.  I'm wondering how I do it without breaking people?  Just republish this thing without a prefix?

**Comment by usepowershell** [comment]
@ChrisAWalker yeah, publish a new version of the module without the c prefixing.. you could even publish a new version of the c prefixed one with a dependency on the new module and have it print a warning that all new development is happening at SecurityOptions rather than cSecurityOptions.
